.azuredevops/appcenter-sdk-unity-build-release-1es.yml

```diff
@@ -13,75 +26,83 @@ resources:
     sdl:
       sourceAnalysisPool: 1ES-PT-Windows-2022
     stages:
     - stage: BuildAppCenterSDKUnity
-      dependsOn: []
       jobs:
       - job: sdkBuildJob
         displayName: BuildAppCenterSDKUnity
         cancelTimeoutInMinutes: 1
         templateContext:
           outputs:
           - output: pipelineArtifact
-            displayName: 'Publish Artifact: framework'
-            path: '$(Build.ArtifactStagingDirectory)'
-            artifactName: assemblies
+            displayName: 'Publish Artifact: UnityPackages'
+            path: 'output'
+            artifactName: UnityPackages
         steps:
+        - task: AzureKeyVault@2
+          displayName: 'Download KeyValult Variables'
+          inputs:
+            azureSubscription: 'AC - Dev Infra & Build Pool'
+            KeyVaultName: 'mobile-center-sdk'
+            SecretsFilter: 'appcenter-sdk-unity-serial-number-windows,appcenter-sdk-unity-username,appcenter-sdk-unity-password,appcensk-appcenter-token,appcenter-sdk-blob-storage-access-secret'
+            RunAsPreJob: false
         - task: PowerShell@2
           displayName: 'Install Unity'
           inputs:
             targetType: filePath
             filePath: ./build.ps1
             arguments: '-Target="Install-Unity-Windows" -Verbosity="Diagnostic"'
-
         - task: UseDotNet@2
           displayName: 'Use .Net Core sdk 3.0.100'
           inputs:
             version: 3.0.100
-
         - task: PowerShell@2
           displayName: 'Activate Unity'
           inputs:
             targetType: filePath
             filePath: ./build.ps1
-            arguments: '-Target="RegisterUnity" -UnitySerialNumber="$(UNITY_SERIAL_NUMBER_WINDOWS)" -UnityUsername="$(UNITY_USERNAME)" -UnityPassword="$(UNITY_PASSWORD)" -Verbosity="Diagnostic"'
-
+            arguments: '-Target="RegisterUnity" -UnitySerialNumber="$(appcenter-sdk-unity-serial-number-windows)" -UnityUsername="$(appcenter-sdk-unity-username)" -UnityPassword="$(appcenter-sdk-unity-password)" -Verbosity="Diagnostic"'
         - script: |
-          echo After activating unity, some important .meta files are deleted, so they must be un-deleted
-          git config --global user.email "you@example.com"
-          git config --global user.name "Your Name"
-          git stash
+            echo After activating unity, some important .meta files are deleted, so they must be un-deleted
+            git config --global user.email "you@example.com"
+            git config --global user.name "Your Name"
+            git stash
           displayName: 'git stash'
-
+# Removing unnecessary file to prevent Binskim failure. This file used for tests and not required for build.
+        - script: |
+            git submodule init
+            git submodule update
+            rm BreakpadSupport/breakpad-support/src/main/cpp/google-breakpad/src/tools/windows/dump_syms/testdata/dump_syms_regtest64.exe
+          displayName: 'git submodule'
```

Comment:
Let's use more human readable display names, eg "Fetch submodules".
Removing the file can be done in a separate task with its own title.